tools/benchmarksAnalyzer/src/main/kotlin/main.kt

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2010-2018 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import org.jetbrains.benchmarksAnalyzer.readFile
+import org.jetbrains.benchmarksAnalyzer.SummaryBenchmarksReport
+import org.jetbrains.report.BenchmarksReport
+import org.jetbrains.report.json.JsonTreeParser
+
+fun printError(message: String) {
+    val usageDesc = "Usage: benchComparator main_report report_to_compare_to [-o output_file] [-eps eps_value] [-short]\n"
+    error("${usageDesc}${message}")
+}
+
+fun main(args: Array<String>) {
+    var mainReport: String? = null
+    var compareToReport: String? = null
+    var outputFile: String? = null
+    var eps: Double = 0.5
```

Comment:
No need for Double type, https://github.com/JetBrains/kotlinconf-spinner/blob/master/getopt/src/main/kotlin/kliopt/KliOpt.kt could be reused.